src/extension.ts

```diff
@@ -246,10 +255,18 @@ export async function attach(
                     DevToolsPanel.createOrShow(context, telemetryReporter, selection.detail, runtimeConfig);
                 }
             }
-        } else {
-            telemetryReporter.sendTelemetryEvent('command/attach/error/no_json_array', telemetryProps);
         }
     } while (useRetry && Date.now() - startTime < timeout);
+
+    // If there is no after the timeout then throw an exception
+    if (!responseArray) {
+        void ErrorReporter.showErrorDialog({
+            errorCode: ErrorCodes.Error,
+            title: 'Error while attaching to target',
```

Comment:
This seems a bit misleading. Won't this hit if we cant get the available targets? "Error while fetching list of available targets"